tests/mock/test_mock_session_gone.c

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017-2019 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#include <string.h>
+#include "../utils/test_setup.h"
+#include "../utils/mock_setup.h"
+
+void test_session_gone_during_query(void **unused) {
```

Comment:
Don't ignore the error while running query.

Reply:
Yea I don't ignore the error while running a query. The purpose of this test is to ensure that if this occurs when we run a query, then we return an error to the user. See below for when we check for an error.

Reply:
oh, sorry about that. I took wrong.